File: project.c
#include <stdio.h>
#include <stdlib.h>
int list();
void addition(int a,int b);
void subtraction(int a,int b);
void multiplication(int a,int b);
void divison(int a,int b);
int main() {
    int a;
    printf("Enter the first number : ");
    scanf("%d",&a);
    int b;
    printf("Enter the second number : ");
    scanf("%d",&b);
    while(1) {
        switch(list()) {
            case 1:
                addition(a,b);
                break;
            case 2:
                subtraction(a,b);
                break;
            case 3:
                multiplication(a,b);
                break;
            case 4:
                divison(a,b);
                break;
            case 5:
                exit(0);
            default :
                printf("You entered Wrong Choice");
        }
        int n;
        printf("Enter 0 for Exit and 1 for Main Menu : ");
        scanf("%d",&n);
        if(n==1) list();
        else exit(0);
    }
    return 0;
}
int list() {
    int ch;
    printf("\nMain Menu : ");
    printf("\n1.Additon");
    printf("\n2.Subtraction");
    printf("\n3.Multiplication");
    printf("\n4.Divison");
    printf("\n5.Exit\n\n");
    printf("Enter your choice : ");
    scanf("%d",&ch);
    return(ch);
}
void addition(int a,int b) {
    printf("Addition = %d\n",a+b);
}
void subtraction(int a,int b) {
    printf("Subtraction = %d\n",a-b);
}
void multiplication(int a,int b) {
    printf("Multiplication = %d\n",a*b);
}
void divison(int a,int b) {
    printf("Divison = %d\n",a/b);
}

File: project.java
import java.util.*;
public class project {
    static int balance = 10000;
        static int menu(){
            Scanner sc = new Scanner(System.in);
            System.out.println("----------Welcome to ATM----------");
            System.out.println();
            System.out.println("Now insert your card in ATM machine");
            System.out.println();
            System.out.println("Menu:");
            System.out.println("1. Withdraw");
            System.out.println("2. Deposit");
            System.out.println("3. Check Balance");
            System.out.println("4. Exit");
            System.out.println();
            System.out.print("Enter your choice : ");
            int ch = sc.nextInt();
            System.out.println();
            return(ch);
        }
        static void withdraw() {
            Scanner sc = new Scanner(System.in);
            System.out.println("Withdrawn Operation:");
                System.out.print("Enter the withdrawing amount : ");
                int with = sc.nextInt();
                if(balance>=with) {
                    balance = balance - with;
                System.out.println("Please collect your money and collect the card");
                System.out.println();
                System.out.println("Balance : " + balance);
                System.out.println();
                System.out.println("Thanks for using ATM !\n\n");
            }
            else {
                System.out.println("Sorry! Insufficient Funds");
                System.out.println();
                System.out.println("Thanks for using ATM !\n\n");
            }
    }
    static void deposite() {
        Scanner sc = new Scanner(System.in);
        System.out.println("Deposit Operation:"); 
            System.out.print("Enter the depositing amount : ");
            int dept = sc.nextInt();
            balance = balance + dept;
            System.out.println("Your Money has been successfully deposited");
            System.out.println();
            System.out.println("Balance : " + balance);
            System.out.println();
            System.out.println("Thanks for using ATM !\n\n");
    }
    static void current() {
        System.out.println("Current Balance : " + balance);
            System.out.println();
            System.out.println("Thanks for using ATM !\n\n");
    }
    public static void main(String[] args) {
        while(true) {
        switch(menu()) {
            case 1:
            withdraw();
            break;
            case 2:
            deposite();
            break;
            case 3:
            current();
            break;
            case 4:
            return;
            default :
            System.out.println("You entered wrong key\n\n");
        }
        System.out.println("Enter the 0 for Exit and 1 for Main Menu : ");
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        if(a==1) menu();
        else return;
    }
    }
}